tests/sqlparser_snowflake.rs

```diff
@@ -2846,3 +2846,32 @@ fn test_parse_show_columns_sql() {
     snowflake().verified_stmt("SHOW COLUMNS IN TABLE abc");
     snowflake().verified_stmt("SHOW COLUMNS LIKE '%xyz%' IN TABLE abc");
 }
+
+#[test]
+fn test_sf_double_dot_notation() {
```

Comment:
can we add a test case containing multiple double dots? e.g. `select * from x, y..z..w`